packages/nepal-scan-scheduler/src/types/scanWindowContinuousPeriodMonthly.ts

```diff
@@ -0,0 +1,23 @@
+/**
+ * ScanScheduler Service
+ * Advanced Scan Scheduling API
+ *
+ * OpenAPI spec version: 2.0
+ * Contact: support@alertlogic.com
+ *
+ * NOTE: This class is auto generated by the swagger code generator program.
+ * https://github.com/swagger-api/swagger-codegen.git
+ * Do not edit the class manually.
+ */
+import { ScanWindow } from './scanWindow';
+
+/**
+ * Scan window for continuous period between two days of month, with start time/day and end time/day.
+ */
+export interface ScanWindowContinuousPeriodMonthly extends ScanWindow {
+    type: ScanWindow.TypeEnum;
```

Comment:
this should be initialised with a concrete value, depending on the class that inherits from the abstract one (ScanWindow).
I think I sorted that out here: 

https://algithub.pd.alertlogic.net/defender/scan_scheduler/commit/f64435ab0d8a71d206e855ba4956c1c577487a3d#diff-e487e0021bb79573e1160ab9db2ebc76R282

Please, let's try to re-generate these files with the updated spec.